_sidebar.md

```diff
@@ -31,6 +31,7 @@
   - [2.5.3 - Deliverables](2-virtual-machines-containers/2.5.3-deliverables.md)
 * **Chapter 3**
 - [3.0 - Cloud Computing](3-cloud-computing/3.0-overview.md)
+  - [3.0.1 - Cloud Service Ecosystems](3-cloud-computing/3.0.1-Cloud Service Ecosystems.md)
```

Comment:
If you decide to keep 3.0.1, due to the spaces in the file name "3.0.1-Cloud Service Ecosystems.md" it is causing it to error out a bit.  I attached a photo of the section in the sidebar below.

<img width="281" alt="Screenshot 2023-02-17 at 11 27 19 AM" src="https://user-images.githubusercontent.com/60158119/219770690-620fb239-066d-44d3-a955-0efa0f612211.png">

Reply:
changed filename and removed spaces